client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -41,12 +45,32 @@ class SharingButtons extends Component {
 	};
 
 	saveChanges = event => {
+		const {
+			isJetpack,
+			isLikesModuleActive,
+			siteId
+		} = this.props;
+
 		event.preventDefault();
+
 		this.props.saveSiteSettings( this.props.siteId, this.state.values );
 		if ( this.state.buttonsPendingSave ) {
 			this.props.saveSharingButtons( this.props.siteId, this.state.buttonsPendingSave );
 		}
 		this.props.recordGoogleEvent( 'Sharing', 'Clicked Save Changes Button' );
+
+		if ( ! isJetpack || isLikesModuleActive !== false ) {
```

Comment:
Yes - we could have just `isLikesModuleActive`, but `!== false` is more precise - it will skip the case when we still don't know what's the Likes module activation status.

Just to clarify, when we don't know the Likes module activation status yet, this can be because the request hasn't loaded yet, or there was a request network failure. In that case, `isLikesModuleActive` will be `null`, and we want to avoid activating the module in that case. We only want to activate the module if we're sure that it's not activated.